.gitignore

```diff
@@ -11,3 +11,4 @@ coverage.xml
 htmlcov/
 /.env
 .envrc
+.vscode
```

Comment:
nit: configure your IDE to leave a blank line at the end of files, for consistency with existing code.

Reply:
Done. The blank line was created.
Thanks!